lib/utils/forkedblockchain.js

```diff
@@ -122,30 +116,20 @@ ForkedBlockchain.prototype.createGenesisBlock = function(callback) {
   });
 };
 
-ForkedBlockchain.prototype.createForkedStorageTrie = function(address) {
-  address = to.hex(address);
-
-  var trie = new ForkedStorageTrie(this.data.trie_db, null, {
-    address: address,
-    stateTrie: this.stateTrie,
-    blockchain: this,
-    fork: this.fork,
-    forkBlockNumber: this.forkBlockNumber
-  });
-
-  this.storageTrieCache[address] = trie;
-
-  return trie;
-};
-
-ForkedBlockchain.prototype.lookupStorageTrie = function(address, callback) {
-  address = to.hex(address);
-
-  if (this.storageTrieCache[address] != null) {
-    return callback(null, this.storageTrieCache[address]);
-  }
+ForkedBlockchain.prototype.getLookupStorageTrie = function(stateTrie, lookupAccount) {
+  lookupAccount = lookupAccount || this.getLookupAccount(stateTrie);
```

Comment:
Reason for change: `getLookupStorageTrie` is usually called when the ephemeralVm is created; when that happens, we already have a `lookupAccount` fn generated, so we used that. There are other ways to call getLookupStorageTrie and those don't have (easy) access to a valid `lookupAccount` fn. So sometimes we have to create it.